OpenBudgeteer.Blazor/Pages/Transaction.razor

```diff
@@ -183,8 +183,17 @@
                             </tr>
                         }
                     </table>
+                                    
```

Comment:
Does it make sense to add this whole logic also to the "Create new Transaction" section (line 47)